website/docs/d/database_users.html.markdown

```diff
@@ -68,6 +67,8 @@ In addition to all arguments above, the following attributes are exported:
 * `roles` - List of user’s roles and the databases / collections on which the roles apply. A role allows the user to perform particular actions on the specified database. A role on the admin database can include privileges that apply to the other databases as well. See [Roles](#roles) below for more details.
 * `database_name` - The user’s authentication database. A user must provide both a username and authentication database to log into MongoDB. In Atlas deployments of MongoDB, the authentication database is always the admin database.
```

Comment:
needs same change as data source for database_users.

Reply:
Thanks, @themantissa, good catch! 👍